content/admin/word_admin.py

```diff
@@ -55,7 +55,7 @@ class WordAdmin(ReviewableAdminMixin, GeneralContentAdmin):
                     'get_definitions', 'get_set_list_display']
     list_filter = [('is_done', admin.BooleanFieldListFilter),
                    ('word_set__name', MultiSelectFieldListFilter)]
-    readonly_fields = ('get_set_list_display',)
+    readonly_fields = ('get_set_list_display', 'memory_aid_highlight')
```

Comment:
`memory_aid_highlight` shouldn't be added here. This admin displays the related characters under a word, and there is no need to show the `memory_aid_highlight` of every character when editing a word.

Reply:
You should add memory_aid_highlight to `WordAdmin`